_Express/views/listing.jade

```diff
@@ -0,0 +1,10 @@
+extends layout
+
+block content 
+ div.listing_display.
+   <img src=#{listingsJSON.listings[id-1].image_url} style="width:300px"><br>
+   Title: #{listingsJSON.listings[id-1].title}<br>
+   Desc: #{listingsJSON.listings[id-1].description}<br>
+   Capacity: #{listingsJSON.listings[id-1].capacity}<br>
+   Price: #{listingsJSON.listings[id-1].price} gold pieces<br>
+   Rating: #{'💀'.repeat(listingsJSON.listings[id-1].rating)}<br><br>
```

Comment:
To keep this DRY, could specify `listingJSON = data.listings[id-1]` on line 9 of `/routes/listing.js` and use `listingJSON` in the view?